mako.5.scd

```diff
@@ -150,6 +157,13 @@ Empty lines and lines that begin with # are ignored.
 
 	Default: 1
 
+*history* = 0|1
```

Comment:
Making this allows to disable history selectively for some notifications with criteria, nice.